src/js/angular/core/directives/queryeditor/query-editor.directive.js

```diff
@@ -48,6 +48,10 @@ function queryEditorDirective($timeout, $location, toastr, $repositories, Sparql
         scope.nostatus = attrs.hasOwnProperty('nostatus');
         // Doesn't show the run button
         scope.norun = attrs.hasOwnProperty('norun');
+
+        scope.columnResizeWidth = scope.$eval(attrs.columnResizeWidth);
```

Comment:
What is the meaning of this `columnResizeWidth`? As I can see from your changes in the code in the other project, this is the minimum window width on which the collumns can be resizable in the table renderer in YASR. Right? IMO the name of this variable is quite misleading as well as the parameter passed to the YASR. Also see the comment in the other MR as well.

Reply:
I will rename it.